### PR TITLE
Update test/e2e manifests

### PR DESCRIPTION
**Pull request by ingvagabund**
- update crd + cr
- fix the sa asset path

### CODE REVIEW

**Comment by ardaguclu** [comment]
I think that https://github.com/openshift/run-once-duration-override-operator/pull/7#discussion_r1092825878 is the reason of this test failed?

**Comment by ingvagabund** [comment]
The `ActiveDeadlineSeconds` field is tested right after the operator starts running. Imho, the admission webhook does not have enough time to start running.

**Comment by ingvagabund** [comment]
Blocked until https://github.com/openshift/run-once-duration-override/pull/8 merges and the image is promoted.

**Comment by ingvagabund** [comment]
Based on the CI cluster the webhooks are running:
```
$ oc get pods
NAME                                                   READY   STATUS    RESTARTS   AGE
run-once-duration-override-operator-77458cfc79-hz2qg   1/1     Running   0          108m
runoncedurationoverride-6drrh                          1/1     Running   0          108m
runoncedurationoverride-7bcgw                          1/1     Running   0          108m
runoncedurationoverride-tfn6h                          1/1     Running   0          108m
```

File: test/e2e/operator_test.go [hunk 1]
```diff
@@ -184,28 +191,97 @@ func TestMain(m *testing.M) {
...
+		return true, nil
+	}); err != nil {
+		klog.Errorf("Unable to collect master nodes")
+		os.Exit(1)
+	}
+
+	// Wait until the webhook pods are running
+	if err := wait.PollImmediate(5*time.Second, 3*time.Hour, func() (bool, error) {
+		klog.Infof("Listing pods...")
+		podItems, err := kubeClient.CoreV1().Pods("run-once-duration-override").List(ctx, metav1.ListOptions{})
```
Comment (ingvagabund) [inline test/e2e/operator_test.go:219]: s/run-once-duration-override/run-once-duration-override-operator

File: test/e2e/operator_test.go [hunk 2]
```diff
@@ -184,28 +191,97 @@ func TestMain(m *testing.M) {
...
+		if err != nil {
+			klog.Errorf("Unable to list pods: %v", err)
+			return false, nil
+		}
+		for _, pod := range podItems.Items {
+			if !strings.HasPrefix(pod.Name, "run-once-duration-override-") {
```
Comment (ingvagabund) [inline test/e2e/operator_test.go]: s/run-once-duration-override-/runoncedurationoverride-/

**Comment by ingvagabund** [comment]
The installer failed
/retest

**Comment by ardaguclu** [comment]
According to the PR https://github.com/openshift/run-once-duration-override-operator/pull/9, this PR should work. Thus;

/retest-required

**Comment by ardaguclu** [comment]
Since the installer phase failed;

/retest-required

**Comment by ardaguclu** [comment]
I think, we can decrease timeout value 3 hours. But this is not a blocker for this PR;

/lgtm

**Pull request merged**